Jenkinsfile

```diff
@@ -10,6 +10,11 @@ elifePipeline {
 
             stage 'Build image', {
                 sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml build"
+                image = DockerImage.elifesciences(this, 'api-dummy', commit)
```

Comment:
wrapping the whole `stage` content in https://github.com/elifesciences/pattern-library/blob/develop/Jenkinsfile#L43-L52 may be useful to understand whether the image build is failing